packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -482,12 +488,13 @@ export default class ComponentsTableRoute extends Route {
   </p>
   <p class="dummy-paragraph">There are a few critical items for developers to note:
     <ul>
-      <li>The table row element (<code class="dummy-code">tr</code>) is not eligible to receive interactions. If an
-        interactive element is desired, place it within a table cell element (<code class="dummy-code">td</code>).</li>
+      <li>The table row element (<code class="dummy-code">tr</code>) is not eligible to receive interactions (e.g.,
+        cannot have an onClick event handler attached directly to it). If an interactive element is desired, place it
+        within a table cell element (<code class="dummy-code">td</code>).</li>
       <li>When providing additional or alternative styles to the table element, do not change the
         <code class="dummy-code">display</code>
         property in the CSS. This alters how the table is presented to the user with assistive technology, and they will
-        no longer be presented with a table.
+        no longer be presented with a table, as the table element will have lost its implicit role.
```

Comment:
Question: Would it be helpful to include a reference link here? To me, it seems non-intuitive that altering a visual display property would affect semantics so it seems like a reference could help HDS users to better understand.

I also wondered if we could possibly add aria roles to the various table elements in case the display property were changed? (Or would it be more harmful to include redundant aria roles for tables using the default "table" display?) I found this article which recommends adding aria roles if you change the display for example: https://www.accessibility-developer-guide.com/examples/tables/layout-changes/

Reply:
Hmm, maybe- I have brought it up before but the idea was discarded. Perhaps it's time to try again? 

I could also revert this specific change- maybe I made it more complicated by trying to clarify. I don't _want_ developers reaching for aria-roles if they don't have to.

WDYT?

Reply:
Ok, I've edited the text for now, hopefully reduces potential confusion.